### DISCUSSION TITLE
Call glb and usdz dynamically

### DISCUSSION THREAD

**Discussion by Arumugamsix** [google/model-viewer#4129] [Q&A]
How to call the src (.glb, .usdz) dynamically from API or another method
<model-viewer src="models/window.gltf" 
<model-viewer src="models/window.usdz"

**Comment by elalish** [comment]
I'm not sure what you mean - can you give me detail?

**Comment by danilokrulj** [comment]
Maybe something like this?

modelViewer.setAttribute('src', "model.gltf");
modelViewer.setAttribute('ios-src', "model.usdz");
